### ISSUE TITLE
Adjust colors for different K

### ISSUE THREAD

**Issue by WalterDurka** [royfrancis/pophelper#14]
Hi!
When plotting for several values of K ("plotQ(slist[2:6]..."), the colors of clusters do not match. 
Imaginge cluster 1 from K=2 stays the same but cluster 2 from K=2 is devided into two clusters at K=3. 
However, the cluster identities are not maintained between Ks. Thus old cluster 1 (from K=2) is cluster 2 at K=3 etc.
I would thus like to keep the same color for those clusters that are the same across Ks.
This means I would like to use different color schemes for the different files of slist in the plotQ command.

Now I would have to re-order the coloums of the files in slist, which is laborious...

Any idea? Or can  you adjust plotQ accordingly?
Best regards
Walter

**Comment by royfrancis** [comment]
Hi Walter,
When the barplots are plotted, colours are always plotted in the same order of clusters as in the input file. But, within run files, the clusters are randomly arranged across repeats and across K, . For example:

Lets says K=4, has 4 runs. Order of clusters would look like below:

Run 1:  C2 C3 C1 C4
Run 2:  C1 C2 C3 C4
Run 3:  C2 C1 C4 C2
Run 4:  C4 C3 C4 C2 

Since the order of clusters differ from run to run, the colours associated with the clusters change too. Reordering clusters automatically is not trivial. In fact there is a software called CLUMPP which orders clusters between runs/repeats within a single K. CLUMPP has been integrated within pophelper and can be run as below:

```r
# read run files
slist <- readQ(sfiles)
# plot in original order only runs with K=3
plotQ(slist[c(3,4,10)],imgoutput="join")

# run this function to order clusters
clumppExport(qlist=slist, useexe=T)

# then read in aligned file for K=3
aligned <- readQ("./pop_K3/pop_K3-combined-aligned.txt")

# plot aligned file
plotQ(aligned,imgoutput="join")
```

See [5.6 clumppExport](http://royfrancis.github.io/pophelper/#56_clumppexport) section in the guide.

Aligning clusters across multiple K is trickier because as K increases, one K splits into two. 

K=3
Run 1:  C2 C3 C1 C4
Run 2:  C1 C2 C3 C4

K=5
Run 1:  C2 C5 C1 C4 C3
Run 2:  C3 C2 C5 C4 C1

Aligning across multiple K is implemented in [CLUMPAK](http://clumpak.tau.ac.il/). This has not been implemented in pophelper yet. You have to upload your run files to the CLUMPAK server.

**Comment by royfrancis** [comment]
Aligning across K implemented in v2.3.0
